### PR TITLE
Update string representation of yasa.Hypnogram

### LINKED ISSUE #128: Change string representation of the yasa.Hypnogram class (__repr__)
@remrama I was thinking that to avoid any confusion, we could change the string representation of the `yasa.Hypnogram` class to *not* print the pandas.Series but instead some general statistics, _a la_ [mne.Raw](https://github.com/mne-tools/mne-python/blob/e5086fcb20e891ef1c4ddc2ce2ae16047b496c5f/mne/io/base.py#L1733)

```python
def __repr__(self):
    # return f"{self.hypno}"
    return (
        f"<yasa.Hypnogram | A {self.duration} minutes hypnogram, with {self.n_stages} stages and {self.freq} resolution>\n"
        " - Use `.hypno` to get the values as a pandas.Series"
    )
```

which gives:

image

I still think that I prefer the original output as a shortcut to the pandas.Series, but would be ready to change if you feel strongly about this new approach.

### PR DESCRIPTION
**Pull request by raphaelvallat**
Closes #128 

Let me know if the text looks good. Happy to make edits.

image

### CODE REVIEW

File: yasa/hypno.py [hunk 1]
```diff
@@ -83,6 +83,15 @@ class Hypnogram:
     >>> from yasa import Hypnogram
     >>> values = ["W", "W", "W", "S", "S", "S", "S", "S", "W", "S", "S", "S"]
     >>> hyp = Hypnogram(values, n_stages=2)
+    >>> hyp
+    <Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages>
+     - Use `.hypno` to get the values as a pandas.Series
+     - Use `.as_int()` to get the values as integers
```
Comment (remrama) [inline yasa/hypno.py:89]: Maybe detail to help differentiate, something like:

- `Use .hypno to get the string values as a pandas.Series`
- `Use .as_int() to get the integer values as a pandas.Series`

File: yasa/hypno.py [hunk 2]
```diff
@@ -83,6 +83,15 @@ class Hypnogram:
     >>> from yasa import Hypnogram
     >>> values = ["W", "W", "W", "S", "S", "S", "S", "S", "W", "S", "S", "S"]
     >>> hyp = Hypnogram(values, n_stages=2)
+    >>> hyp
+    <Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages>
```
Comment (remrama) [inline yasa/hypno.py:87]: Maybe an if statement to add scorer if present:

- if None: `<Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages>`
- if YASA: `<Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages, scored by YASA>`

This one I'm not sure about so do whatever you think is best.

**Review by remrama — APPROVED** [review]
Approved 💃

File: yasa/hypno.py (hunk 2 above)
Comment (raphaelvallat) [reply yasa/hypno.py:87]: Good idea, done!

**Pull request merged**